pom.xml

```diff
@@ -13,33 +13,38 @@
     <name>psc-delta-consumer</name>
     <description>The psc-delta-consumer is responsible for transforming psc data from the psc-delta kafka topic as part of chips and chs data sync.</description>
     <properties>
-        <java.version>11</java.version>
+        <java.version>21</java.version>
         <main.class>uk.gov.companieshouse.psc.delta.PscDeltaConsumerApplication</main.class>
-        <spring-boot-dependencies.version>2.6.4</spring-boot-dependencies.version>
-        <spring-boot-maven-plugin.version>2.6.4</spring-boot-maven-plugin.version>
+        <spring-boot-dependencies.version>3.3.5</spring-boot-dependencies.version>
+        <spring-boot-maven-plugin.version>3.3.5</spring-boot-maven-plugin.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
-        <maven-build-helper-plugin.version>3.2.0</maven-build-helper-plugin.version>
-        <maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
-        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-        <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-        <jib-maven-plugin.version>3.3.0</jib-maven-plugin.version>
-        <ch-kafka.version>1.4.2</ch-kafka.version>
-        <structured-logging.version>1.9.25</structured-logging.version>
-        <kafka-models.version>1.0.34</kafka-models.version>
-        <private-api-sdk-java.version>2.0.446</private-api-sdk-java.version>
-        <api-sdk-manager-java-library.version>1.0.4</api-sdk-manager-java-library.version>
-        <map-struct.version>1.5.3.Final</map-struct.version>
-        <api-helper-java-library.version>1.4.5</api-helper-java-library.version>
-        <maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
+        <maven-build-helper-plugin.version>3.6.0</maven-build-helper-plugin.version>
+        <maven-failsafe-plugin.version>3.5.2</maven-failsafe-plugin.version>
+        <maven-compiler-plugin.version>3.13.0</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>3.5.2</maven-surefire-plugin.version>
+        <maven-checkstyle-plugin.version>3.6.0</maven-checkstyle-plugin.version>
+        <jib-maven-plugin.version>3.4.4</jib-maven-plugin.version>
+        <ch-kafka.version>3.0.3</ch-kafka.version>
+        <structured-logging.version>3.0.20</structured-logging.version>
```

Comment:
[Probably needs the starter-logging dependency](https://companieshouse.atlassian.net/wiki/spaces/JU/pages/4901634070/StructuredLogger+format+missing+context)

Reply:
done